From 2fc02ab237c2008344538966b44ffd4b1a4b4676 Mon Sep 17 00:00:00 2001
From: Yasufumi Ogawa <yasufum.o@gmail.com>
Date: Tue, 27 Aug 2019 18:37:22 +0900
Subject: [PATCH] tools/sppc: change suricata ubuntu latest

Sometimes, building suricata on Ubuntu 18.04 is failed even if using
tar.gz file, and it is more stable to build as 16.04 even though steps
for building is increased.

Signed-off-by: Yasufumi Ogawa <yasufum.o@gmail.com>
---
 .../build/ubuntu/suricata/Dockerfile.18.04    | 36 +++++++++----------
 .../build/ubuntu/suricata/Dockerfile.latest   | 36 +++++++++----------
 2 files changed, 36 insertions(+), 36 deletions(-)

diff --git a/tools/sppc/build/ubuntu/suricata/Dockerfile.18.04 b/tools/sppc/build/ubuntu/suricata/Dockerfile.18.04
index e48d873..7648736 100644
--- a/tools/sppc/build/ubuntu/suricata/Dockerfile.18.04
+++ b/tools/sppc/build/ubuntu/suricata/Dockerfile.18.04
@@ -74,29 +74,29 @@ WORKDIR ${rte_sdk}/${rte_target}
 RUN make -j 4
 
 # NOTE: Case 1
-WORKDIR ${home_dir}
-RUN wget ${SURICATA_DOWNLOAD}
-RUN tar zxvf ${SURICATA_TGZ}
-WORKDIR ${home_dir}/${SURICATA_SRCDIR}
-RUN RTE_SDK=${rte_sdk} RTE_TARGET=${rte_target} autoconf
-RUN RTE_SDK=${rte_sdk} RTE_TARGET=${rte_target} sh configure --enable-dpdk
-RUN make -j 10
-ENV PATH ${SURICATA_SRCDIR}/src:${PATH}
+#WORKDIR ${home_dir}
+#RUN wget ${SURICATA_DOWNLOAD}
+#RUN tar zxvf ${SURICATA_TGZ}
+#WORKDIR ${home_dir}/${SURICATA_SRCDIR}
+#RUN RTE_SDK=${rte_sdk} RTE_TARGET=${rte_target} autoconf
+#RUN RTE_SDK=${rte_sdk} RTE_TARGET=${rte_target} sh configure --enable-dpdk
+#RUN make -j 10
+#ENV PATH ${SURICATA_SRCDIR}/src:${PATH}
 
 # NOTE: Case 2
 # Compiling suricata with Dockerfile is failed if clone suricata
 # while building this image. So, do it after app container is launched.
 # Create suricata install script.
-#WORKDIR ${home_dir}
-#RUN echo "#!/bin/bash" > ${home_dir}/${INSTALL_SURICATA_SH}
-#RUN echo "" >> ${home_dir}/${INSTALL_SURICATA_SH}
-#RUN echo "git clone ${suricata_repo}" >> ${home_dir}/${INSTALL_SURICATA_SH}
-#RUN echo "cd ${suricata_dir}" >> ${home_dir}/${INSTALL_SURICATA_SH}
-#RUN echo "autoconf" >> ${home_dir}/${INSTALL_SURICATA_SH}
-#RUN echo "./configure --enable-dpdk" >> ${home_dir}/${INSTALL_SURICATA_SH}
-#RUN echo "make -j 10" >> ${home_dir}/${INSTALL_SURICATA_SH}
-#RUN echo "make install" >> ${home_dir}/${INSTALL_SURICATA_SH}
-#RUN chmod 775 ${home_dir}/${INSTALL_SURICATA_SH}
+WORKDIR ${home_dir}
+RUN echo "#!/bin/bash" > ${home_dir}/${INSTALL_SURICATA_SH}
+RUN echo "" >> ${home_dir}/${INSTALL_SURICATA_SH}
+RUN echo "git clone ${suricata_repo}" >> ${home_dir}/${INSTALL_SURICATA_SH}
+RUN echo "cd ${suricata_dir}" >> ${home_dir}/${INSTALL_SURICATA_SH}
+RUN echo "autoconf" >> ${home_dir}/${INSTALL_SURICATA_SH}
+RUN echo "./configure --enable-dpdk" >> ${home_dir}/${INSTALL_SURICATA_SH}
+RUN echo "make -j 10" >> ${home_dir}/${INSTALL_SURICATA_SH}
+RUN echo "make install" >> ${home_dir}/${INSTALL_SURICATA_SH}
+RUN chmod 775 ${home_dir}/${INSTALL_SURICATA_SH}
 
 # NOTE: Case 3 Compiling suricata with Dockerfile does not work
 # Compile suricata
diff --git a/tools/sppc/build/ubuntu/suricata/Dockerfile.latest b/tools/sppc/build/ubuntu/suricata/Dockerfile.latest
index c3928a0..67e3380 100644
--- a/tools/sppc/build/ubuntu/suricata/Dockerfile.latest
+++ b/tools/sppc/build/ubuntu/suricata/Dockerfile.latest
@@ -74,29 +74,29 @@ WORKDIR ${rte_sdk}/${rte_target}
 RUN make -j 4
 
 # NOTE: Case 1
-WORKDIR ${home_dir}
-RUN wget ${SURICATA_DOWNLOAD}
-RUN tar zxvf ${SURICATA_TGZ}
-WORKDIR ${home_dir}/${SURICATA_SRCDIR}
-RUN RTE_SDK=${rte_sdk} RTE_TARGET=${rte_target} autoconf
-RUN RTE_SDK=${rte_sdk} RTE_TARGET=${rte_target} sh configure --enable-dpdk
-RUN make -j 10
-ENV PATH ${SURICATA_SRCDIR}/src:${PATH}
+#WORKDIR ${home_dir}
+#RUN wget ${SURICATA_DOWNLOAD}
+#RUN tar zxvf ${SURICATA_TGZ}
+#WORKDIR ${home_dir}/${SURICATA_SRCDIR}
+#RUN RTE_SDK=${rte_sdk} RTE_TARGET=${rte_target} autoconf
+#RUN RTE_SDK=${rte_sdk} RTE_TARGET=${rte_target} sh configure --enable-dpdk
+#RUN make -j 10
+#ENV PATH ${SURICATA_SRCDIR}/src:${PATH}
 
 # NOTE: Case 2
 # Compiling suricata with Dockerfile is failed if clone suricata
 # while building this image. So, do it after app container is launched.
 # Create suricata install script.
-#WORKDIR ${home_dir}
-#RUN echo "#!/bin/bash" > ${home_dir}/${INSTALL_SURICATA_SH}
-#RUN echo "" >> ${home_dir}/${INSTALL_SURICATA_SH}
-#RUN echo "git clone ${suricata_repo}" >> ${home_dir}/${INSTALL_SURICATA_SH}
-#RUN echo "cd ${suricata_dir}" >> ${home_dir}/${INSTALL_SURICATA_SH}
-#RUN echo "autoconf" >> ${home_dir}/${INSTALL_SURICATA_SH}
-#RUN echo "./configure --enable-dpdk" >> ${home_dir}/${INSTALL_SURICATA_SH}
-#RUN echo "make -j 10" >> ${home_dir}/${INSTALL_SURICATA_SH}
-#RUN echo "make install" >> ${home_dir}/${INSTALL_SURICATA_SH}
-#RUN chmod 775 ${home_dir}/${INSTALL_SURICATA_SH}
+WORKDIR ${home_dir}
+RUN echo "#!/bin/bash" > ${home_dir}/${INSTALL_SURICATA_SH}
+RUN echo "" >> ${home_dir}/${INSTALL_SURICATA_SH}
+RUN echo "git clone ${suricata_repo}" >> ${home_dir}/${INSTALL_SURICATA_SH}
+RUN echo "cd ${suricata_dir}" >> ${home_dir}/${INSTALL_SURICATA_SH}
+RUN echo "autoconf" >> ${home_dir}/${INSTALL_SURICATA_SH}
+RUN echo "./configure --enable-dpdk" >> ${home_dir}/${INSTALL_SURICATA_SH}
+RUN echo "make -j 10" >> ${home_dir}/${INSTALL_SURICATA_SH}
+RUN echo "make install" >> ${home_dir}/${INSTALL_SURICATA_SH}
+RUN chmod 775 ${home_dir}/${INSTALL_SURICATA_SH}
 
 # NOTE: Case 3 Compiling suricata with Dockerfile does not work
 # Compile suricata
-- 
2.17.1

